app/src/main/java/de/danoeh/antennapod/activity/PreferenceActivity.java

```diff
@@ -128,7 +131,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     @Override
     public void onSearchResultClicked(SearchPreferenceResult result) {
-        PreferenceFragmentCompat fragment = openScreen(result.getResourceFile());
-        result.highlight(fragment);
+        int screen = result.getResourceFile();
+        switch (screen) {
+            case R.xml.feed_settings:
+                AlertDialog.Builder builder = new AlertDialog.Builder(this);
+                builder.setTitle(R.string.feed_settings_label);
+                builder.setMessage("This setting is specific to each podcast. It can be accessed " +
```

Comment:
Please use a string resource (core/src/main/res/values/strings.xml), so it can be translated.

Reply:
Ok i'll make new request soon with the new changes.

Reply:
You don't need to make a new request. You can commit and push changes to this existing PR :)

Reply:
Oh, ok. Still new to contributing. I'll note that. So do you approve of the PR?